EncoreTickets.SDK/Venue/Services/VenueAuthenticationServiceApi.cs

```diff
@@ -0,0 +1,39 @@
+﻿using RestSharp;
+
+namespace EncoreTickets.SDK.Venue
+{
+    /// <inheritdoc />
+    /// <summary>
+    /// The authentication service for a venue service.
+    /// </summary>
+    public class VenueAuthenticationServiceApi : BaseCapabilityServiceApi
```

Comment:
in my view, I think that the authentication module should be a generic service by itself. it takes in a service host and login URI and context (which includes a method, in here it would be JWT). it uses username and password to fetch a JWT token and returns it or appends the context.
the VenueServiceApi would then use this Auth service module and call the authenticate with its context, credentials and method (JWT). if the method jwt is confusing, it is just a future concept to make the authenticate method future-proof but we can ignore in the beginning

Reply:
Sounds logical, I'll fix it

Reply:
this is what I mostly care about in the PR :) plz let me know when you do this change